src/frontend/screens/WebView/index.tsx

```diff
@@ -124,6 +125,14 @@ function WebView() {
     window.api.trackScreen('WebView', { url: startUrl, runner })
   }, [startUrl, runner])
 
+  useEffect(() => {
+    window.api.handleGoToGamePage(async (_, gameId) => {
+      const gameInfo = await getGameInfo(gameId, 'hyperplay')
+      navigate(`/gamepage/hyperplay/${gameId}`, { state: { gameInfo, fromDM: false } })
+    }
+    )
+  }, [])
```

Comment:
I don't understand this change 🤔

Why do we need to navigate to the game page when on mount? This component is used for multiple webviews

```tsx
<Route
    path="hyperplaystore"
    element={<WebView key="hyperplaystore" />}
/>
<Route path="epicstore" element={<WebView key="epicstore" />} />
<Route path="gogstore" element={<WebView key="gogstore" />} />
<Route path="wiki" element={<WebView key="wiki" />} />
<Route path="metamaskHome" element={<MetaMaskHome />} />
<Route
    path="metamaskSnaps"
    element={<WebView key="metamaskSnaps" />}
/>
```

Reply:
Not really, this just adds a listener on mount.
Otherwise when we call this method from the store it won't work.

So basically when we mount the Webview we start listening to these calls.

But that is true that will be for all URLs, I will add a check to only add the listener for hyperplay store.